invisible_cities/cities/components.py

```diff
@@ -902,10 +908,13 @@ def calculate_and_save_buffers(buffer_length    : float        ,
                                      saved_buffers                ))
 
     fn_list = (find_signal      ,
+               filter_events_signal,
+               fl.branch(write_signal_filter),
```

Comment:
Generally speaking, lining the commas up vertically helps to avoid mistakes and misunderstandings, and this can be especially important in dataflow pipelines. Now, adjusting the commas at the end of the line is a PITA, and adds unnecessary noise to the diffs.

I recommend that we adopt the following style. Rather than

``` python
fn(foo,
   baaaaaaaa,
   baz,
   quux,
   quuuuuux)
```
where comma-errors or surprises are difficult to spot, or 
``` python
fn(foo      ,
   baaaaaaaa,
   baz      ,
   quux     ,
   quuuuuux )
```
which is a PITA to keep aligned, do this:
``` python
fn( foo
  , baaaaaaaa
  , baz
  , quux
  , quuuuuux)
```
This will really freak out the PEP8-ists, (and many of you) but, turst me, it's the best way forward :-)